README.md

```diff
@@ -41,7 +41,7 @@ val text = env.socketTextStream(host, port, '\n')
 val windowCounts = text.flatMap { w => w.split("\\s") }
   .map { w => WordWithCount(w, 1) }
   .keyBy("word")
-  .timeWindow(Time.seconds(5))
+  .window(TumblingEventTimeWindow.of(Time.seconds(5)))
```

Comment:
I think here it will not work without a `WatermarkStrategy`, right? If this is correct, we should maybe put processing time window.